assignments/objects.js

```diff
@@ -18,28 +18,65 @@ const example = {
 }
 
 // Write your intern objects here:
+const Mitzi = {
```

Comment:
Syntax is very easy to read and clean.  Great job creating objects and returning the data required.